README.md

```diff
@@ -215,15 +215,53 @@ drop messages or crash, also based on configuration.
 Subscriptions are managed using `eredis_sub:subscribe/2` and
 `eredis_sub:unsubscribe/2`. When Redis acknowledges the change in
 subscription, a message is sent to the controlling process for each
-channel.
+channel. Then, eredis sends a message on the form `{subscribed, Channel,
+ClientPid}` to the controlling process, which must be acked using
+`eredis_sub:ack_message/1`.
 
 eredis also supports Pattern Subscribe using `eredis_sub:psubscribe/2`
-and `eredis_sub:unsubscribe/2`. As with normal subscriptions, a message
-is sent to the controlling process for each channel.
+and `eredis_sub:punsubscribe/2`. As with normal subscriptions, a message
+is sent to the controlling process for each channel and eredis sends a
+message on the form `{subscribed, Channel, ClientPid}` to the
+controlling process, which must be acked using `eredis_sub:ack_message/1`.
+
+The controlling process is also notified in case of reconnection attempts or
+failures. See `test/eredis_pubsub_SUITE.erl` for examples.
+
+Starting with version 1.5, eredis automatically resubscribes after reconnect.
```

Comment:
Since we dont have a config for this new behavior it would be good to describe it in CHANGELOG.md.
Maybe there are other good places for the info? possibly on the release page on github for them that reads that.